spec/lib/pdf_filler/md502_pdf_spec.rb

```diff
@@ -0,0 +1,145 @@
+require 'rails_helper'
+
+RSpec.describe PdfFiller::Md502Pdf do
+  include PdfSpecHelper
+
+  let(:intake) { create(:state_file_md_intake) }
+  let(:submission) { create :efile_submission, tax_return: nil, data_source: intake }
+  let(:pdf) { described_class.new(submission) }
+
+  describe '#hash_for_pdf' do
+    let(:file_path) { described_class.new(submission.reload).output_file.path }
+    let(:pdf_fields) { filled_in_values(file_path) }
+
+    it 'uses field names that exist in the pdf' do
+      missing_fields = pdf.hash_for_pdf.keys.map { |k| k.to_s.gsub("'", "&apos;").to_s } - pdf_fields.keys
```

Comment:
oh this looks new! was there an issue with this particular PDF or something?

Reply:
the apostrophes in the field name gets turned into `&apos;` in the `hash_to_pdf` method